Makefile

```diff
@@ -57,3 +57,11 @@ fix: misspell-correction
 install-tools: $(MISSPELL)
 	npm install
 	@echo "All tools installed"
+
+.PHONY: build-docker-images
+build-docker-images:
+	docker compose -f compose.yml build
```

Comment:
Should we rename `compose.yml` to `docker-compose.yml` which appears to be the recommended file name that by default is used instead of having to specify with the `-f` option

Reply:
Actually the `docker-compose.yml`is the old version of it:
https://docs.docker.com/compose/compose-file/#compose-file

It's still supported just for backward compatibility.
With the `compose.yml` we do not need to specify the `-f` when running.

Reply:
It's not a blocker though. we are good to merge it!

Reply:
Merging now